aws/ec2/test_ec2_instance_has_required_tags.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from aws.ec2.helpers import ec2_instance_test_id
+from aws.ec2.resources import ec2_instances
+
+
+@pytest.fixture
+def required_tag_names(pytestconfig):
+    return frozenset(pytestconfig.getoption('--aws-require-tag'))
```

Comment:
We could move this to conftest.py if other tests need it.

Alternatively the test could use `pytestconfig` directly, but having it as as separate fixture might make caching it easier.